docs/_pages/0980-release-management.md

```diff
@@ -270,16 +270,14 @@ development cycle.
 
 ### 10. Update Kubernetes Manifests
 
-Based on the generated helm charts run the [./hack/update-k8s-manifests.sh][update-script]
+Based on the generated helm charts run `make k8s-manifests-update VERSION=<version>` target
```

Comment:
Please correct me if I am wrong, I am not sure if the placement of the word 'target' is right. Thanks.
Otherwise, LGTM

Reply:
I was trying to imply run k8s-manifests-update make target.